t/valgrind/workload.d/job-multinode

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
+
+NJOBS=${NJOBS:-10}
+
+flux submit --cc="1-$NJOBS" --wait hostname
```

Comment:
Missing `-N2` here?

Also, I'm not sure it is necessary to run 10 2 node jobs, since this will extend the time the testsuite runs, and multiple jobs is already testing in the jobs workload. Maybe 2 jobs should be the default here?

Reply:
oh crud ... i was going back and forth changing it ...